code/datums/emotes.dm

```diff
@@ -130,13 +130,8 @@
 
 	//SKYRAT EDIT ADDITION BEGIN - AI QoL
 	for(var/mob/ai in GLOB.ai_list)
```

Comment:
```suggestion
	for(var/mob/living/silicon/ai/ai as anything in GLOB.ai_list)
```

Reply:
I'ma be real I actually don't know the difference between using as anything or not but the last time I had to do smth like this I was told to use as anything so there